README.md

```diff
@@ -22,9 +22,13 @@ Or install it yourself as:
 
 ## Usage
 
+### Fill watermark image method
+
+`fill_watermark_image(img, width, height, watermark_image_filename, gravity = "Center", horizontal_margin = 0, vertical_margin = 0, opacity = 100)`
+
 Just like any other Refile processor, for example:
 
-    <%= image_tag attachment_url(@user, :profile_image, :fill_watermark_image, 300, 300, "watermark.jpg", format: "jpg") %>
+    <%= image_tag attachment_url(@user, :profile_image, :fill_watermark_image, 300, 300, "watermark.jpg", "SouthEast", 10, 10, 50) %>
```

Comment:
Will the `format: "jpg"` options still work? I want to make sure this change is backwards-compatible and won't break anyone's installation.